onadata/libs/serializers/attachment_serializer.py

```diff
@@ -54,7 +54,7 @@ class AttachmentSerializer(serializers.HyperlinkedModelSerializer):
     class Meta:
         fields = ('url', 'filename', 'mimetype', 'field_xpath', 'id', 'xform',
                   'instance', 'download_url', 'small_download_url',
-                  'medium_download_url')
+                  'medium_download_url', 'deleted_at')
```

Comment:
is this exposing `deleted_at` to users? This should be handled opaquely, ie I don't see how/why a user needs to see this ever, in an edge-case an dev might need to see this but I'd rather wait to understand why we'd expose this to a user before doing so